cpu/cc26xx-cc13xx/Makefile.cc26xx-cc13xx

```diff
@@ -6,7 +6,15 @@ OBJCOPY = arm-none-eabi-objcopy
 OBJDUMP = arm-none-eabi-objdump
 NM      = arm-none-eabi-nm
 SIZE    = arm-none-eabi-size
+SREC_TEST = $(shell srec_cat --version > null && echo 1)
+ifeq ([${SREC_TEST}],[1])
 SREC_CAT = srec_cat
+CAN_HEX  = %.hex
+else
+$(waring "install srecord to allow hex image generation")
+SREC_CAT =
```

Comment:
`SREC_CAT` doesn't need to be defined conditionally.  It's make-rule simply won't be used if `CAN_HEX` is empty.  Therefore, it always can be defined as `srec_cat`; and, it can be used in the `$(shell)` test.